spec/support/corpus.rb

```diff
@@ -139,8 +139,7 @@ def checkout
       #
       # @return [Integer] mutations generated
       def check_generation(path)
-        node = Parser::CurrentRuby.parse(path.read)
-        fail "Cannot parse: #{path}" unless node
+        node = Parser::CurrentRuby.parse(path.read) or return 0
```

Comment:
That exception was kinda stupid, as `parser` returns `nil` on empty `.rb` files, which is "unfortunate" but valid `.rb` files.